index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+<body>
+    <header class="header">
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+                <i class="material-symbols-outlined menu_icon">
+                    menu
+                </i>
+            </nav>
+    </header>
+    <div class="container">
+        <h2>I’m Abdurahim Glad to see you!</h2>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="social_media">LET’S CONNECT</p>
```

Comment:
- [ ] This is the sub-heading so I would suggest you please use `h2, h3, h4` tag for better semantic HTML.